packages/components/molecules/f-tabs/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v3.4.0
+------------------------------
+*March 27, 2023*
```

Comment:
I hope you haven't been waiting this long for a review 😆 

```suggestion
*May 24, 2023*
```

Reply:
lol will update the date